src/components/input/gamepadtokey.js

```diff
@@ -254,101 +254,93 @@ require(['apphost'], function (appHost) {
     function runInputLoop() {
         // Get the latest gamepad state.
         var gamepads;
-        if (navigator.getGamepads) {
-            gamepads = navigator.getGamepads();
-        } else if (navigator.webkitGetGamepads) {
-            gamepads = navigator.webkitGetGamepads();
-        }
-        gamepads = gamepads || [];
+        gamepads = navigator.getGamepads();
         var i;
         var j;
         var len;
         for (i = 0, len = gamepads.length; i < len; i++) {
             var gamepad = gamepads[i];
-            if (gamepad) {
```

Comment:
I just tested on PC. I have Xbox Wireless Adapter, and it always reports 4 gamepads, even only one is connected. So this condition is required.

Reply:
@dmitrylyzo This is why [I made this loop](https://github.com/jellyfin/jellyfin-web/pull/746/files#diff-56b9612ccb19d654f1b875a6598c51a0R129-R141) in ``keyboardnavigation.js``. The ``gamepadlist`` will always return 4 ``gamepad`` objects. All of them will be empty if no gamepad is connected. However, they will be populated once a gamepad is connected.

Afaik, the problem is in ``keyboardnavigation.js``, I will describe it to you in your comment there.